src/main/java/com/aws/greengrass/logmanager/model/LogFileGroup.java

```diff
@@ -35,7 +35,7 @@ public static LogFileGroup create(Pattern filePattern, URI path, Instant lastUpd
         File folder = new File(path);
 
         if (!folder.isDirectory()) {
-            throw new InvalidLogGroupException("Must be a folder.");
+            throw new InvalidLogGroupException(String.format("%s must be a directory", path));
```

Comment:
what if it doesn't exist, but it _will_ exist in the future and will be a directory? What would we do then?

Reply:
The creation of logFileGroup is like a snapshot-mechanism to be created in every cycle. 
If the directory doesn't exist now, we could just skip it for this cycle. 
But if it exist in the next round, the logFileGroup will be newly created, so it will become a valid directory.

Reply:
OK, that's good. What do we do with the exception? We shouldn't be continuously logging the same issue. Definitely not at error level.

Reply:
Agree that it should not be at error Level, I will put it in the debug level.